tests/appsec/test_processor.py

```diff
@@ -135,3 +135,26 @@ def test_appsec_span_tags_snapshot(tracer):
         set_http_meta(span, {}, raw_uri="http://example.com/.git", status_code="404")
 
     assert "triggers" in json.loads(span.get_tag("_dd.appsec.json"))
+
+
+def test_appsec_span_rate_limit(tracer):
+    with override_env(dict(DD_APPSEC_TRACE_RATE_LIMIT="1")):
+        _enable_appsec(tracer)
+
+        # we have 2 spans going through with a rate limit of 1: this is because the first span will update the rate
+        # limiter last update timestamp. In other words, we need a first call to reset the rate limiter's clock
+        with tracer.trace("test", span_type=SpanTypes.WEB) as span1:  # aligning rate limiter clock with this span (this
+            # span will go through as it is linked to the init window)
```

Comment:
The way this comment is broken onto two lines is weird (for python at least).
```suggestion
        # DEV: aligning rate limiter clock with this span (this
        #      span will go through as it is linked to the init window)
        with tracer.trace("test", span_type=SpanTypes.WEB) as span1:
```

?